vochain/scrutinizer/process.go

```diff
@@ -37,8 +37,18 @@ func (s *Scrutinizer) ProcessList(entityID []byte, fromID []byte, max int64) ([]
 		if !fromLock {
 			keyCopy := make([]byte, len(process))
 			copy(keyCopy, process)
-			processListResult = append(processListResult, keyCopy)
-			max--
+			if p, err := s.VochainState.Process(keyCopy, true); err != nil {
```

Comment:
I don't like this approach, is very expensive. For each ProcessList query we'll be fetching all processes from the state...
I'd rather try to save on the processList scrutinizer storage the namespace. So splinting by bytes (reserving N bytes for the namespace) would make the trick.

Reply:
agree, this is much more efficient